crates/compiler/test_mono/generated/lambda_capture_niches_with_non_capturing_function.txt

```diff
@@ -1,22 +1,22 @@
 procedure Test.1 (Test.5):
-    let Test.19 : [C , C U64, C {}] = TagId(2) Test.5;
+    let Test.19 : [C , C {}, C U64] = TagId(1) Test.5;
```

Comment:
The deltas here are due to how we sort multimorphic lambda sets (lambda sets with multiple functions of the same name but different captures). We first sort them by name, then by capture sets. The sorting for capture sets has changed since captures are now always interned, whereas they were layouts before. The change is safe though.